android/src/main/java/com/thebluealliance/androidclient/Utilities.java

```diff
@@ -1,31 +1,36 @@
 package com.thebluealliance.androidclient;
 
+import com.google.common.base.Predicate;
+
+import com.thebluealliance.androidclient.activities.GamedayActivity;
```

Comment:
We need to figure out how we want autoformat to sort imports and make everyone set that as the default so we don't keep getting these changes